submissions/evgeniy241984/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,102 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+"use strict";
+
+class Animals {
+  constructor(gender, name, saying) {
+    this.gender = gender;
+    this.name = name;
+    this.saying = saying;
+  }
+
+  say() {
+    return `i'm ${this.species}; My name is: <strong> ${this.name} </strong>; My gender: ${this.gender}; Saying: <em> ${this.saying} </em>`;
```

Comment:
Current implementation of `Animals` doesn't provide `species` property, child classes do. Parent class **should not* know anything about any child's props.